core/modules/proto/TaskMsgDigest.cc

```diff
@@ -27,6 +27,10 @@
 // System headers
 #ifdef __SUNPRO_CC
 #include <sys/md5.h>
+#elif __APPLE__
+#define COMMON_DIGEST_FOR_OPENSSL
+#include <CommonCrypto/CommonDigest.h>
+#define MD5 CC_MD5
```

Comment:
Line 44 of StringHash.cc below?  I will pull the branch and look more closely so I can see what's going on...